hack/release-from/README.md

```diff
@@ -0,0 +1,52 @@
+# release-from
+
+`release-from` is a tool designed to streamline the process of creating a new operator
+using a previously released operator version.
+
+The base operator version must reference either a tag or commit hash in `tigera/operator`
+
+## Installation
+
+To install `release-from`, use the following command:
+
+```bash
+make hack/bin/release-from
+```
+
+## Usage
+
+To start, familarize yourself with the tool
+
+```sh
+release-from --help
+```
+
+To create a new release
+
+```sh
+release-from --base-version <previous operator version> --version <version to release> \
+  [--except-calico | --except-calico-enterprise] <image>:<image version>
+```
+
+> [!IMPORTANT]
+> To publish the newly created operator, use the `--publish` flag
```

Comment:
Maybe you can clarify what publish means, from what I understand it is pushing the tag, pushing images + image manifest, but not creating a release in https://github.com/tigera/operator/releases